dace/frontend/python/replacements/definitions.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2019-2024 ETH Zurich and the DaCe authors. All rights reserved.
+"""
+Contains definitions of new data containers (arrays, locals, streams)
```

Comment:
I guess this is array creation. We can opt to say data/container creation/manipulation to make it more dace-like.